src/Documents/Changes/DatabaseChanges.ts

```diff
@@ -378,6 +398,12 @@ export class DatabaseChanges implements IDatabaseChanges {
             const messages = Array.isArray(payloadParsed) ? payloadParsed : [payloadParsed];
             for (const message of messages) {
                 const type = message.Type;
+                if (message.TopologyChange) {
+                    this._getOrAddConnectionState("Topology", "watch-topology-change", "", "");
+                    // noinspection ES6MissingAwait
+                    this._requestExecutor.updateTopology(this._serverNode, 0, true, "watch-topology-change");
```

Comment:
Why no `await` here? Where would it be awaited? It should be somewhere I think?

Reply:
to match java code:
https://github.com/ravendb/ravendb-jvm-client/blob/v4.2/src/main/java/net/ravendb/client/documents/changes/DatabaseChanges.java#L568